phy/stats/ccg.py

```diff
@@ -0,0 +1,133 @@
+# -*- coding: utf-8 -*-
+
+"""Cross-correlograms."""
+
+#------------------------------------------------------------------------------
+# Imports
+#------------------------------------------------------------------------------
+
+import numpy as np
+
+from ..ext import six
+from ..utils.array import _index_of, _unique
+
+
+#------------------------------------------------------------------------------
+# Cross-correlograms
+#------------------------------------------------------------------------------
+
+def _increment(arr, indices):
+    """Increment some indices in a 1D vector of non-negative integers.
```

Comment:
assert that the indices aren't out of range? i.e. that indices doesn't contain something more than len(arr)?